emacs.d/b-lsp.el

```diff
@@ -0,0 +1,19 @@
+(provide 'b-lsp)
+
+(require 'package)
+(setq package-user-dir (concat (file-name-directory load-file-name) "melpa"))
+(add-to-list 'package-archives '("melpa" . "https://melpa.org/packages/") t)
+(package-initialize)
+
+(require 'lsp-mode)
+(require 'lsp-pyright)
+(setq lsp-completion-provider :none)
+(setq lsp-pyright-auto-import-completions
+      nil)
+(setq lsp-signature-auto-activate nil) ;; disalbe bottom documentation popup
+(setq company-dabbrev-downcase 1)
+(setq company-idle-delay 0.01)
+(add-hook 'js-mode-hook #'lsp)
+(add-hook 'web-mode-hook #'lsp)
+(setq gc-cons-threshold 100000000)
```

Comment:
definitely in bivio-init.el

Reply:
Should `read-process-output-max` go over too? How do you decide what you put in bivio-init.el vs here?

Reply:
ditto.